packages/support/test/unit/image-util.spec.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable promise/prefer-await-to-callbacks */
+/* eslint-disable require-await */
+import {createSandbox} from 'sinon';
+import rewiremock from 'rewiremock/node';
+
+const {expect} = chai;
```

Comment:
cannot we just import it?

Reply:
we could. but it's already in the global context (as to avoid the boilerplate of `import chai from 'chai'`) everywhere, so ..

Reply:
my point is that explicit is better than implicit. We don't know what's in the global context by just reading the source here. I'd rather prefer having one extra line

Reply:
sorry... if we start adding `import whatever from 'chai'` to test files, the convention becomes inconsistent, and for consistency we might as well avoid all of the globals.  I value the reduction of boilerplate more than any FUD around the assignment and referencing of global variables; that's a conscious decision.